src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,93 @@
+package duke.parser;
+
+import duke.command.*;
```

Comment:
Not sure I like the use of wildcard imports here. Perhaps you could checkout the settings of your IDE to ensure it does not collapse the import statements into a wildcard automatically for you.